src/actions/biblioActions.js

```diff
@@ -36,18 +36,6 @@ export const changeFieldArrayReferenceJson = (e) => {
   };
 };
 
-export const deleteFieldModReferenceReferenceJson = (e) => {
```

Comment:
keep

Reply:
Good catch!!! @azurebrd For some reason, I have a wrong version of biblioActions.js to start with. Just rebased code from main. So the missing code should be back in place now.

Reply:
Great, thanks !